### PR TITLE
lint `./examples/*`

### PR DESCRIPTION
**Pull request by jGaboardi**
This PR lints and re-formats the `./examples/*` directory. --> xref #597

### CODE REVIEW

File: libpysal/examples/base.py [hunk 1]
```diff
@@ -9,24 +9,23 @@
...
 
 def get_data_home():
-    """Return the path of the ``libpysal`` data directory. This folder is platform specific.
-    If the folder does not already exist, it is automatically created.
+    """Return the path of the ``libpysal`` data directory. This folder is platform
+    specific. If the folder does not already exist, it is automatically created.
 
     Returns
     -------
+
```
Comment (martinfleis) [inline libpysal/examples/base.py:28]: Where this empty line comes from?

**Review by martinfleis — COMMENTED** [review]
One question on docstring formatting, otherwise looks great!

File: libpysal/examples/base.py (hunk 1 above)
Comment (jGaboardi) [reply libpysal/examples/base.py:28]: Those were added by me. I prefer them for readability and ease of maintenance.

Comment (martinfleis) [reply libpysal/examples/base.py:28]: Could we just stick to the numpydoc style guide? https://numpydoc.readthedocs.io/en/latest/format.html

Not that I mind the empty lines but if there's no strong reason to deviate, I would generally try to stick to consistent style we can refer to.

Comment (jGaboardi) [reply libpysal/examples/base.py:28]: Yes, I can revert. Actually,  I had confused myself about the numpydoc style. I had thought it included spaces. Maybe `ruff`'s next endeavor will be a proper doc formatter & linter.

Comment (jGaboardi) [reply libpysal/examples/base.py]: Done.

**Review by martinfleis — APPROVED** [review]

**Pull request merged**